docs/05-infrastructure-nodes.md

```diff
@@ -0,0 +1,318 @@
+# OpenShift Infrastructure Nodes
+The OpenShift subscription model allows customers to run various core
+infrastructure components at no additional charge. In other words, a node
+that is only running core OpenShift infrastructure components is not counted
+in terms of the total number of subscriptions required to cover the
+environment.
+
+OpenShift components that fall into the infrastructure categorization
+include:
+
+* kubernetes and OpenShift control plane services ("masters")
```

Comment:
Are you sure, we already have control plane nodes for this.

Reply:
exactly that. you don't get charged for the masters. you wouldn't, for example, run ES+Kibana on a master. So you'd have masters+ES+Kibana hosts/nodes. "masters" (control plane services) are included in the subscription.

I'm totally open to better/different wording here!